### PR TITLE
Addition of test_computeresource_azure.py module with azure CR stubs

### PR DESCRIPTION
**Pull request by san7ket**
Added some test stubs for tier1, tier2 , tier3
Added the new file to rst to create documentation.

### CODE REVIEW

**Comment by renzon** [comment]
ACK

File: tests/foreman/ui/test_computeresource_azure.py [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+        2. Provide a valid subscription ID and a path
+           to the ".pem" certificate.
+        3. Select the created azure CR and click images tab.
+        4. Select "New image" , provide it valid name and information.
+        5. Select the desired template to create image and submit.
+
+        @Assert: The image is added to the CR successfully
+
+        @Caseautomation: notautomated
+         """
```
Comment (rochacbruno) [inline tests/foreman/ui/test_computeresource_azure.py]: nitpick, but looks like you have indent problem is that closing line.

Comment (san7ket) [inline tests/foreman/ui/test_computeresource_azure.py]: fixed.

**Comment by sghai** [comment]
ACK
@kbidarkar : hey could you please review these tests once ?

File: tests/foreman/ui/test_computeresource_azure.py [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+
+        @id: 6cd42f2f-44cb-4e10-b5f9-bcc617b63cab
+
+        @setup:
+
+        1. Management certificates, two certificates are needed
+        2. ".cer" file, which is uploaded to Azure.
+        3. ".pem" file, which is stored locally.
+        4. Subscription ID from azure.
+        5. Images added in azure.
```
Comment (kbidarkar) [inline tests/foreman/ui/test_computeresource_azure.py]: This needs to be, "Images available in azure region."

Comment (kbidarkar) [inline tests/foreman/ui/test_computeresource_azure.py]: Please update all these instances.

**Comment by kbidarkar** [comment]
ACK, with small update to setup step for images.

Some info on my comment.
For cloud CR's, images are mostly available and we can use them.
For Virtualization CR's, we need to create the images manually.

**Comment by kbidarkar** [comment]
Feel free to merge this PR once, the comments are addressed.

**Comment by san7ket** [comment]
Comment's resolved.

**Comment by kbidarkar** [comment]
ACK, all comment's resolved

**Pull request merged**